routers/routes/routes.go

```diff
@@ -869,9 +869,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 
 					m.Get("/edit", repo.EditProject)
 					m.Post("/edit", bindIgnErr(auth.CreateProjectForm{}), repo.EditProjectPost)
+					m.Put("/issue/priority", bindIgnErr(auth.UpdateIssuePriorityBoardForm{}), repo.UpdateBoardIssuePriority)
 					m.Post("/^:action(open|close)$", repo.ChangeProjectStatus)
 
 					m.Group("/:boardID", func() {
+						m.Put("board/priority", bindIgnErr(auth.UpdateBoardPriorityForm{}), repo.UpdateBoardsPriorityPost)
```

Comment:
```suggestion
						m.Put("/board/priority", bindIgnErr(auth.UpdateBoardPriorityForm{}), repo.UpdateBoardsPriorityPost)
```

Reply:
@lafriks this doesn't work for me, I get a 401 when I hit `/:user/:repo/projects/:id/board/priority` if I remove the `/` it works though.

Reply:
This is stranage :/